docs/README.FreeBSD.md

```diff
@@ -82,7 +82,7 @@ If you get a `package not found` type of message with the below command, remove
 
 Install build dependencies:
 ```
-sudo pkg install autoconf automake avahi-app binutils cmake curl dbus doxygen e2fsprogs-libuuid enca encodings flac flatbuffers font-util fontconfig freetype2 fribidi fstrcmp gawk gettext-tools giflib git glew gmake gmp gnutls googletest gperf gstreamer1-vaapi hal jpeg-turbo libaacs libass libbdplus libbluray libcapn libcdio libcec libedit libfmt libgcrypt libgpg-error libidn libinotify libmicrohttpd libnfs libogg libplist librtmp libtool libudev-devd libva libvdpau libvorbis libxslt lirc lzo2 m4 mesa-libs mysql57-client nasm openjdk8 p8-platform pkgconf python3 rapidjson shairplay sndio sqlite3 swig30 taglib tiff tinyxml tinyxml2 xf86-input-keyboard xf86-input-mouse xorg-server xrandr zip
+sudo pkg install autoconf automake avahi-app bash binutils cmake curl dbus doxygen e2fsprogs-libuuid enca encodings evdev-proto flac flatbuffers font-util fontconfig freetype2 fribidi fstrcmp gawk gettext-tools giflib git glew gmake gmp gnutls googletest gperf gstreamer1-vaapi jpeg-turbo libaacs libass libbdplus libbluray libcapn libcdio libcec libdisplay-info libedit libfmt libgcrypt libgpg-error libidn libinotify libinput libmicrohttpd libnfs libogg libplist librtmp libtool libudev-devd libva libvdpau libvorbis libxkbcommon libxslt lirc lzo2 m4 mesa-libs nasm openjdk18 p8-platform pcre pkgconf python3 rapidjson shairplay sndio spdlog sqlite3 swig taglib tiff tinyxml tinyxml2 wayland-protocols waylandpp xf86-input-keyboard xf86-input-mouse xorg-server xrandr zip
```

Comment:
why bash?

as you saw pcre is not really recommended on freebsd 14, so id actually look to use -DENABLE_INTERNAL_PCRE=ON as part of the cmake kodi project configuration option rather than install a system pcre.

openjdk, i think id lean more towards an LTS version. openjdk17 or openjdk21.

Reply:
id probably use mariadb-connector-c-3.3.8_1 instead of flat out removing mysql57-client

Reply:
> why bash?

needed to build internal ffmpeg. actually our jenkins is using internal build. we can choose between adding system `ffmpeg` and removing `bash` or vice versa

https://github.com/xbmc/xbmc/blob/0f7f6ac821a31b0360e8563b98b2d435a8c98d80/cmake/modules/FindFFMPEG.cmake#L102

> use -DENABLE_INTERNAL_PCRE=ON

makes perfect sense

> towards an LTS version. openjdk17  or openjdk21

will check with 21

> use mariadb-connector-c-3.3.8_1

yes, i missed that one

Reply:
add ffmpeg as a pkg install in that case. I imagine freebsd is of the same philosopy of the *nixes, use system shared libs

current ffmpeg version on 14 is ffmpeg-6.1.1_5,1, but looks like its the same for 13+ by default.